guiEditor/src/diagram/properties/propertyComponentProps.ts

```diff
@@ -1,7 +1,13 @@
 import { Control } from "babylonjs-gui/2D/controls/control";
 import { GlobalState } from "../../globalState";
+import { GUINode } from "../guiNode";
 
 export interface IPropertyComponentProps {
     globalState: GlobalState;
-    guiControl: Control;
+    guiBlock: Control;

```

Comment:
Did you have a specific reason to rename `guiControl` to `guiBlock`? `guiControl` seems better than `guiBlock` given that the type is `Control` . Or was it too generic?